### ISSUE TITLE
Bookmarks menu: 'remove' and 'rename' are ambiguous

### ISSUE THREAD

**Issue by LinuxOnTheDesktop** [linuxmint/nemo#3433]
### Distribution

Mint 21.3

### Package version

6.0.2

### Frequency

Always

### Bug description

The bookmark context menu has 'remove' and 'rename'. Those labels make the user wonder: remove or rename _the bookmark_, or the bookmarked _items themselves_. Given, especially, that we have, in addition to bookmarks, 'My computer' and 'favourites', I think it important to reduce opportunities for confusion.

### Steps to reproduce

Get up Nemo's bookmark context-menu.

### Expected behavior

'Remove' -> 'Remove bookmark'.
'Rename' -> 'Rename bookmark'.

**Comment by Jeremy7701** [comment]
Why would anyone go to the bookmarks to rename/remove the target of a bookmark?
The clue is in the name - if I remove a bookmark from a book, it doesn't remove any pages!
Rename even allows you enter the new name of the bookmark - in the bookmark menu.

**Comment by LinuxOnTheDesktop** [comment]
@Jeremy7701

That is all true, except that, in short, in a GUI it is easier than with an actual book to forget the type of thing with which one is dealing with. Hopefully this screenshot illustrates:

![image](https://github.com/linuxmint/nemo/assets/47056543/84c2c4a7-6800-46dd-aa4b-c8a34b24e1e3)

Is 'Documents' - which one might be set to remove - a bookmark? Actually, it is a 'My Computer item' (and, thus I don't actually know whether it is a bookmark, or something like a bookmark).

**Comment by Jeremy7701** [comment]
If you delete your Documents folder (or rename it or move it etc) the nemo bookmark will be highlighted (because its invalid) then next time you start nemo and you can delete or fix the bookmark.
Example:-
![Screenshot from 2024-07-11 20-25-16](https://github.com/user-attachments/assets/b6a5d19d-c6f4-40b1-95ed-7f3d2e03b261)

If you "accidentally" delete the bookmark (but not your data!) its easily recreated.

**Comment by LinuxOnTheDesktop** [comment]
@Jeremy7701 

Seemingly there's been some miscommunication here. My most recent point was as follows. The user wonders about the difference between the following two types of item listed in Nemo's right-hand pane: (1) those filed under 'my computer'; (2) those filed under bookmarks.

I made the foregoing point partly for its own sake - can that difference be clarified somehow/somewhere? - and in support of the following idea. _It would indeed be a good idea to make more explicit, within the bookmark content menu, just what the actions in that menu will do._ What is needed on that latter front is to change 'Remove' to 'Remove bookmark' and to change 'Rename' to 'Rename bookmark'. I can create a pull-request to that latter effect, but only if you are good enough to direct me towards the relevant bit of code. Thanks. Please remember that one of Mint's main goals is to be user-friendly.
